pp/integrations/plugin-packs/procedures/infrastructure-active-directory-nrpe.md

```diff
@@ -1,80 +1,283 @@
 ---
 id: infrastructure-active-directory-nrpe
-title: Microsoft Active Directory
+title: Microsoft Active Directory NRPE
 ---
 import Tabs from '@theme/Tabs';
 import TabItem from '@theme/TabItem';
 
+This Monitoring Connector monitors Active Directory Domain Controllers using local commands and the Centreon NSClient++ and the version 2 of the NRPE protocol agent to trigger their executions remotely.
```

Comment:
```suggestion
This Monitoring Connector monitors Active Directory Domain Controllers using local commands and the Centreon NSClient++ and version 2 of the NRPE protocol agent to trigger their executions remotely.
```

Reply:
Tthere is an extra element compared to the French version. What are "local commands"?

Reply:
Commands that are executed locally on the windows server